ani-cli

```diff
@@ -98,7 +105,7 @@ update_script() {
 # checks if dependencies are present
 dep_ch() {
     for dep; do
-        command -v "$dep" >/dev/null || die "Program \"$dep\" not found. Please install it."
+        command -v "${dep%% *}" >/dev/null || die "Program \"$dep\" not found. Please install it."
```

Comment:
:eyes:

Reply:
I thought of a way to make player args using the `ANI_CLI_PLAYER` work however this isn't a complete fix and would probably require posix violations

Reply:
yeah, please remove it then. That has nothing to do with the objective of the PR

Reply:
done